BrainPortal/cbrain_plugins/cbrain-plugins-base/userfiles/text_file/views/_text_file.html.erb

```diff
@@ -22,29 +22,18 @@
 #
 -%>
 
-<%
-  if params[:content_loader] == "collection_file" && params[:arguments]
-    argument_list = params[:arguments]
-    argument_list = [argument_list] unless argument_list.is_a?(Array)
-    path = @userfile.collection_file(*argument_list)
-  else
-    path = @userfile.cache_full_path
-  end
-%>
-
 <% if @userfile.is_locally_synced? %>
-
   <% if params[:content_viewer] == "off" || (@userfile.size || 999_999) < 1000 %>
-    <% content = File.read(path) rescue nil %>
+    <% content = File.read(@userfile.cache_full_path) rescue nil %>
```

Comment:
I don't understand how this can even work. If `@userfile` is one of the fake viewer file, the method cache_full_path will not work at all...